### ISSUE TITLE
Same issue as the other Uconnect addons for Alfa Romeo in the US

### ISSUE THREAD

**Issue by julianrinaldi** [Blueion76/FCAUconnect-HA#3]
I get this response:

```
[11:07:57 DBG] {
  "IdentityId": "us-east-1:2e776fa6-5d8f-4f1c-9427-xxxx",
  "Token": "xxxxxxx"
}
[11:09:56 INF] REFRESH SESSION
[11:09:56 DBG] {
  "CallId": "xxxx",
  "ErrorCode": 0,
  "ErrorDetails": null,
  "ErrorMessage": null,
  "ApiVersion": 2,
  "StatusCode": 200,
  "StatusReason": "OK",
  "Time": "2024-12-07T16:09:56.843+00:00"
}
[11:09:57 DBG] {
  "UID": "MCUSEN_AR2011070008xxxx",
  "SessionInfo": {
    "login_token": "st2.s.AtLtO-xxxxx"
  },
  "CallId": "75dc5ae67998457e9ac82775cdxxxx",
  "ErrorCode": 0,
  "ErrorDetails": null,
  "ErrorMessage": null,
  "ApiVersion": 2,
  "StatusCode": 200,
  "StatusReason": "OK",
  "Time": "2024-12-07T16:09:57.097+00:00"
}
[11:09:57 DBG] {
  "id_token": "xxxx",
  "CallId": "c1c91aa24772494795432fa996xxxx",
  "ErrorCode": 0,
  "ErrorDetails": null,
  "ErrorMessage": null,
  "ApiVersion": 2,
  "StatusCode": 200,
  "StatusReason": "OK",
  "Time": "2024-12-07T16:09:57.299+00:00"
}
[11:09:57 INF] Now fetching new data...
[11:09:57 DBG] {
  "IdentityId": "us-east-1:2e776fa6-5d8f-4f1c-9427-722xxxx",
  "Token": "xxxxxxxx"
}
[11:09:57 DBG] {
  "Userid": "MCUSEN_xxxxxx",
  "Version": 0,
  "Vehicles": []
}
[11:09:57 INF] Fetching COMPLETED. Next update in 2 minutes.
```

This is basically the same response I receive with the other addons. It connects ok, but then doesn't pull any vehicles or information. It is a 2023 Alfa Romeo Giulia Quadrifolgio.

**Comment by Blueion76** [comment]
Just to confirm, you do have an active Uconnect subscription and can view the car in the official app correct? Strange because someone else with a 24 A.R. had it working fine.

**Comment by Blueion76** [comment]
I looked into this a little more and found that your car seems to possibly run some other OS than Uconnect 5 (what most other FCA vehicles use) - this in itself isn't super important but it makes me wonder if the API is modified from other FCA vehicles running mainstream Uconnect 5.

**Comment by julianrinaldi** [comment]
Yes, I have a subscription. It seems to have also been locked out from even connecting. I'll have to wait for a few hours and see if it resets and allows connections again, and I'll change the refresh setting to 10 minutes from the default 2 mins and see if it at least stays connected.

**Comment by julianrinaldi** [comment]
> Strange because someone else with a 24 A.R. had it working fine.

Was it a US based Alfa Romeo? The old add-on seemed to work everywhere but in the US or North America.

**Comment by Blueion76** [comment]
> > Strange because someone else with a 24 A.R. had it working fine.
> 
> 
> 
> Was it a US based Alfa Romeo? The old add-on seemed to work everywhere but in the US or North America.

Pretty sure it was from what I can tell: https://github.com/Blueion76/FCAUconnect-HA/discussions/2#discussioncomment-11491811

**Comment by julianrinaldi** [comment]
> > > Strange because someone else with a 24 A.R. had it working fine.
> > 
> > 
> > Was it a US based Alfa Romeo? The old add-on seemed to work everywhere but in the US or North America.
> 
> Pretty sure it was from what I can tell: [#2 (comment)](https://github.com/Blueion76/FCAUconnect-HA/discussions/2#discussioncomment-11491811)

They seem to have a UK Travel integration in their home assistant configuration, so I'm betting they're in the UK.

https://github.com/DubhAd/Home-AssistantConfig/blob/live/integrations.md

**Comment by Blueion76** [comment]
> > > > Strange because someone else with a 24 A.R. had it working fine.
> 
> > > 
> 
> > > 
> 
> > > Was it a US based Alfa Romeo? The old add-on seemed to work everywhere but in the US or North America.
> 
> > 
> 
> > Pretty sure it was from what I can tell: [#2 (comment)](https://github.com/Blueion76/FCAUconnect-HA/discussions/2#discussioncomment-11491811)
> 
> 
> 
> They seem to have a UK Travel integration in their home assistant configuration, so I'm betting they're in the UK.
> 
> 
> 
> https://github.com/DubhAd/Home-AssistantConfig/blob/live/integrations.md

Some others have reported issues with AlfaRomeo, I'm currently stumped as the endpoints all seem valid. I'll look into it on and off but I don't think there's much I can do currently, not a coding geek by any means haha. If anyone has a solution feel free to post here or make a PR and I'll implement it.

**Comment by travi** [comment]
> This is basically the same response I receive with the other addons. It connects ok, but then doesn't pull any vehicles or information.

I have similar behavior for my Jeep Gladiator. In both the original addon and this one, it has always appeared to connect successfully, but returns no vehicles.

**Comment by nk2261** [comment]
I have a 2024 Jeep Compass and am having the same issue in the US. I connects but never creates the objects in MQTT

**Comment by Blueion76** [comment]
> I have a 2024 Jeep Compass and am having the same issue in the US. It connects but never creates the objects in MQTT

It seems like your vehicle uses SiriusXM Guardian from some quick research. If that's the case, you won't be able to use this repo as it's a completely separate system.

**Comment by Blueion76** [comment]
> It seems like your vehicle uses SiriusXM Guardian from some quick research. If that's the case, you won't be able to use this repo as it's a completely separate system.

It seems like your vehicle uses SiriusXM Guardian from some quick research. If that's the case, you won't be able to use this repo as it's a completely separate system.

**Comment by Blueion76** [comment]
> > This is basically the same response I receive with the other addons. It connects ok, but then doesn't pull any vehicles or information.
> 
> I have similar behavior for my 2023 Jeep Gladiator. In both the original addon and this one, it has always appeared to connect successfully, but returns no vehicles.

It seems like your vehicle uses SiriusXM Guardian from some quick research. If that's the case, you won't be able to use this repo as it's a completely separate system.

**Comment by Blueion76** [comment]
> I get this response:
> 
> ```
> [11:07:57 DBG] {
>   "IdentityId": "us-east-1:2e776fa6-5d8f-4f1c-9427-xxxx",
>   "Token": "xxxxxxx"
> }
> [11:09:56 INF] REFRESH SESSION
> [11:09:56 DBG] {
>   "CallId": "xxxx",
>   "ErrorCode": 0,
>   "ErrorDetails": null,
>   "ErrorMessage": null,
>   "ApiVersion": 2,
>   "StatusCode": 200,
>   "StatusReason": "OK",
>   "Time": "2024-12-07T16:09:56.843+00:00"
> }
> [11:09:57 DBG] {
>   "UID": "MCUSEN_AR2011070008xxxx",
>   "SessionInfo": {
>     "login_token": "st2.s.AtLtO-xxxxx"
>   },
>   "CallId": "75dc5ae67998457e9ac82775cdxxxx",
>   "ErrorCode": 0,
>   "ErrorDetails": null,
>   "ErrorMessage": null,
>   "ApiVersion": 2,
>   "StatusCode": 200,
>   "StatusReason": "OK",
>   "Time": "2024-12-07T16:09:57.097+00:00"
> }
> [11:09:57 DBG] {
>   "id_token": "xxxx",
>   "CallId": "c1c91aa24772494795432fa996xxxx",
>   "ErrorCode": 0,
>   "ErrorDetails": null,
>   "ErrorMessage": null,
>   "ApiVersion": 2,
>   "StatusCode": 200,
>   "StatusReason": "OK",
>   "Time": "2024-12-07T16:09:57.299+00:00"
> }
> [11:09:57 INF] Now fetching new data...
> [11:09:57 DBG] {
>   "IdentityId": "us-east-1:2e776fa6-5d8f-4f1c-9427-722xxxx",
>   "Token": "xxxxxxxx"
> }
> [11:09:57 DBG] {
>   "Userid": "MCUSEN_xxxxxx",
>   "Version": 0,
>   "Vehicles": []
> }
> [11:09:57 INF] Fetching COMPLETED. Next update in 2 minutes.
> ```
> 
> This is basically the same response I receive with the other addons. It connects ok, but then doesn't pull any vehicles or information. It is a 2023 Alfa Romeo Giulia Quadrifolgio.

Hey man, I finally have an answer for you regarding this. Unfortunately, your vehicle seems to be one of two modern AlfaRomeo models that use the SiriusXM Guardian services instead of the Uconnect Mopar services. That system is entirely different, so this integration won't be of much use unless a major code rewrite occurs which is unlikely.

**Comment by travi** [comment]
> > > This is basically the same response I receive with the other addons. It connects ok, but then doesn't pull any vehicles or information.
> > 
> > 
> > I have similar behavior for my 2023 Jeep Gladiator. In both the original addon and this one, it has always appeared to connect successfully, but returns no vehicles.
> 
> It seems like your vehicle uses SiriusXM Guardian from some quick research. If that's the case, you won't be able to use this repo as it's a completely separate system.

unfortunate, but appreciate the investigation and confirmation
